highcharts-behavior.js

```diff
@@ -0,0 +1,266 @@
+import * as async from '@polymer/polymer/lib/utils/async.js'
+
+import 'highcharts/highcharts.js';
+import 'highcharts/highcharts-more.js';
+
+Highcharts.setOptions({global: {useUTC: false}});
+
+// from jQuery
+function isFunction(obj) {
+	typeof obj === "function" && typeof obj.nodeType !== "number"
+}
+
+// from jQuery
+function isPlainObject(obj) {
+	var proto, Ctor;
+	// Detect obvious negatives
+	// Use toString instead of jQuery.type to catch host objects
+	if ( !obj || toString.call( obj ) !== "[object Object]" ) {
+		return false;
+	}
+
+	proto = Object.getPrototypeOf( obj );
+	// Objects with no prototype (e.g., `Object.create( null )`) are plain
+	if ( !proto ) {
+		return true;
+	}
+
+	// Objects with prototype are plain iff they were constructed by a global Object function
+	var hasOwn = {}.hasOwnProperty, fnToString = hasOwn.toString;
+	Ctor = hasOwn.call( proto, "constructor" ) && proto.constructor;
+	return typeof Ctor === "function" && fnToString.call( Ctor ) === fnToString.call( Object );
+}
+
+// from jQuery
+function _extends() {
+	var options, name, src, copy, copyIsArray, clone,
+		target = arguments[ 0 ] || {},
+		i = 1,
+		length = arguments.length,
+		deep = false;
+
+	// Handle a deep copy situation
+	if ( typeof target === "boolean" ) {
+		deep = target;
+
+		// Skip the boolean and the target
+		target = arguments[ i ] || {};
+		i++;
+	}
+
+	// Handle case when target is a string or something (possible in deep copy)
+	if ( typeof target !== "object" && !isFunction(target) ) {
+		target = {};
+	}
+
+	// Extend jQuery itself if only one argument is passed
+	if ( i === length ) {
+		target = this;
+		i--;
+	}
+
+	for ( ; i < length; i++ ) {
+		// Only deal with non-null/undefined values
+		if ( ( options = arguments[ i ] ) == null ) {
+			continue;
+		}
+		// Extend the base object
+		for ( name in options ) {
+			src = target[ name ];
+			copy = options[ name ];
+
+			// Prevent never-ending loop
+			if ( target === copy ) {
+				continue;
+			}
+
+			// Recurse if we're merging plain objects or arrays
+			if ( deep && copy && ( isPlainObject( copy ) ||
+				( copyIsArray = Array.isArray( copy ) ) ) ) {
+
+				if ( copyIsArray ) {
+					copyIsArray = false;
+					clone = src && Array.isArray( src ) ? src : [];
+				} else {
+					clone = src && isPlainObject( src ) ? src : {};
+				}
+				target[ name ] = _extends( deep, clone, copy );
+			} else if ( copy !== undefined ) {
+				target[ name ] = copy;
+			}
+		}
+	}
+
+	return target;
+};
```

Comment:
Delete functions above!
- `_extends`
- `isPlainObject`
And replace with:

```suggestion
const _extend = (...a) => Object.assign({}, ...a)
```